eng/targets/Helix.targets

```diff
@@ -97,6 +97,7 @@
     <PropertyGroup>
       <BuildHelixPayload Condition="'@(_HelixApplicableTargetQueue->Count())' == '0'">false</BuildHelixPayload>
       <BuildHelixPayload Condition="'$(TargetArchitecture)' == 'arm64' AND '$(SkipHelixArm)' == 'true'">false</BuildHelixPayload>
+      <BuildHelixPayload Condition="$(HelixTargetQueue.StartsWith('(Alpine.')) AND '$(SkipHelixAlpine)' == 'true'">false</BuildHelixPayload>
```

Comment:
@BrennanConroy I'm continuing our conversation about the msbuild failure here.

The `<BuildHelixPayload Condition="$(SkipHelixQueues.Contains('$(HelixTargetQueue)'))">false</BuildHelixPayload>` a line below uses `'` in a condition. What are you recommending I change this line to?